.travis.yml

```diff
@@ -14,8 +14,7 @@ env:
   global:
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - DOTNET_CLI_TELEMETRY_OPTOUT: 1
-mono:
-  - 4.0.5
+mono: none
```

Comment:
Right now, nobody is running tests on travis on mono (for > 6 months), if you though you were I'd like to talk to you 😄